Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.7
+
+# ENVS RECOMENDATIONS
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# PREPARE FOLDER
+WORKDIR /api
+
+# COPY DEPENDENCIES
+COPY Pipfile Pipfile.lock ./
+
+# INSTALL DEPENDENCIES
+RUN pip install pipenv
+RUN pipenv install --system --deploy --dev
```

Comment:
We should install from the requirements files once this goes in.  #245

Reply:
Which requirements file? requirements-dev.txt or requirements.txt?

Reply:
Just `requirements.txt`

Reply:
Done!!